eth/api.go

```diff
@@ -1506,17 +1508,58 @@ func NewPrivateDebugAPI(eth *Ethereum) *PrivateDebugAPI {
 	return &PrivateDebugAPI{eth: eth}
 }
 
-// ProcessBlock reprocesses an already owned block.
-func (api *PrivateDebugAPI) ProcessBlock(number uint64) (bool, error) {
+// Block processes the given block's RLP but does not import the block in to
+// the chain.
+func (api *PrivateDebugAPI) Block(blockRlp []byte, logger VmLoggerOptions) (bool, error) {
```

Comment:
I'm not sure it's a good idea to drop the "verb" from the method name. Although I get that `debug.Block` looks fancy, imho it's not clear that debug is namespace and verb at the same time. I'd vote to add back either the process or something `debug.ProcessBlock` `debug.RunBlock` `debug.DebugBlock`. Not sure what is the best, but simple debug.Block doesn't really tell what it does.